### PR TITLE
convert unautorized capabilities tests to pact.io

### PR DESCRIPTION
**Pull request by swoichha**
Convert `unauthorized/capabilities.js` and `unauthorized/groupTest.js` to pact.io
part of #500

### CODE REVIEW

**Review by kiranparajuli589 — COMMENTED** [review]
lgtm :+1:

File: tests/karma.js [hunk 1]
```diff
@@ -556,6 +592,142 @@ beforeAll(function (done) {
...
+            '  <statuscode>997</statuscode>\n' +
+            '  <message>Unauthorised</message>\n' +
+            ' </meta>\n' +
+            ' <data/>\n' +
+            '</ocs>'
+        }
+      }))
+    .then(() =>
+      provider.addInteraction({
+        uponReceiving: 'a unauthorized group GET request',
```
Comment (jasson99) [inline tests/karma.js:631]: I think this and the interaction added below have the same request name.

**Review by individual-it — CHANGES_REQUESTED** [review]

File: tests/karma.js [hunk 2]
```diff
@@ -10,13 +10,19 @@ const Pact = require('@pact-foundation/pact-web')
 
 const accessControlAllowHeaders = 'OC-Checksum,OC-Total-Length,OCS-APIREQUEST,X-OC-Mtime,Accept,Authorization,Brief,Content-Length,Content-Range,Content-Type,Date,Depth,Destination,Host,If,If-Match,If-Modified-Since,If-None-Match,If-Range,If-Unmodified-Since,Location,Lock-Token,Overwrite,Prefer,Range,Schedule-Reply,Timeout,User-Agent,X-Expected-Entity-Length,Accept-Language,Access-Control-Request-Method,Access-Control-Allow-Origin,ETag,OC-Autorename,OC-CalDav-Import,OC-Chunked,OC-Etag,OC-FileId,OC-LazyOps,OC-Total-File-Length,Origin,X-Request-ID,X-Requested-With'
 const origin = 'http://localhost:9876'
-const validAuthHeaders = { authorization: 'Basic ' + validUserPasswordHash, Origin: origin }
+const validAuthHeaders = {
```
[the remark this answers is not in the document]
Comment (swoichha) [reply tests/karma.js:13]: No, it was automatically done by IDE

File: tests/groupTest.js [hunk 3]
```diff
@@ -1,4 +1,4 @@
-fdescribe('Main: Currently testing group management,', function () {
+describe('Main: Currently testing group management,', function () {
```
Comment (jasson99) [inline tests/groupTest.js:1]: why is this?

Comment (swoichha) [reply tests/groupTest.js]: fixed

**Review by individual-it — APPROVED** [review]

**Pull request merged**